MapifyEditor/Export/Validators/Validator.cs

```diff
@@ -35,9 +35,10 @@ public static IEnumerator<Result> Validate()
             Scenes scenes = Scenes.FromEnumerable(sceneStates.Keys.ToList());
             for (int i = 0; i < validators.Length; i++)
             {
-                EditorUtility.DisplayProgressBar("Validating map", null, i / (float)validators.Length);
-
                 Validator validator = validators[i];
+
+                EditorUtility.DisplayProgressBar("Validating map", validator.GetType().ToString(), i / (float)validators.Length);
```

Comment:
What's with this change?

Reply:
I ran into a situation where part of the validation took a long time. This line makes the progress bar show what validator is running so we can see what it's doing.

Reply:
It was already there through, you just moved it below the array access

Reply:
Notice the second argument to `EditorUtility.DisplayProgressBar`